bundles/org.eclipse.ui.workbench.texteditor/src/org/eclipse/ui/internal/findandreplace/overlay/FindReplaceOverlay.java

```diff
@@ -353,6 +353,10 @@ public boolean close() {
 
 	@Override
 	public int open() {
+		super.open();
```

Comment:
This results in multiple calls to `super.open()`, as it is already called in a subsequent if block. Why is an additional call necessary? At least, I would only expect one of the calls to be necessary.